dataland-internal-storage/src/main/kotlin/org/dataland/datalandinternalstorage/services/DatabaseStringDataStore.kt

```diff
@@ -176,6 +178,18 @@ class DatabaseStringDataStore(
         logger.info("Inserting data into database with data ID: $dataId and correlation ID: $correlationId.")
         storeDataItemWithoutTransaction(DataItem(dataId, objectMapper.writeValueAsString(data)))
         publishStorageEvent(payload, correlationId)
+
+        val bypassQa = JSONObject(payload).getBoolean("bypassQa")
```

Comment:
Why are we using two different JSON libraries (`org.json` and `jackson`). Let's use jackson for everything to keep it consistent. Even better: Let's use jackson to parse the payload into a kotlin class and operate on that :)

Reply:
I will contact you directly